### PR TITLE
Settings - set name, bio and social links and backup seed phrase

### PR DESCRIPTION
**Pull request by Valentina1133**
https://ethstatus.testrail.net/index.php?/cases/view/703006
https://ethstatus.testrail.net/index.php?/cases/view/703001
Screenshot 2023-09-20 at 19 46 43

### CODE REVIEW

**Review by anastasiyaig — APPROVED** [review]

File: gui/screens/settings.py [hunk 1]
```diff
@@ -24,27 +28,37 @@ class LeftPanel(QObject):
...
-    def _open_settings(self, index: int):
-        self._settings_section_template.real_name['objectName'] = f'{index}-AppMenuItem'
+    def _open_settings(self, index: int, section_name: str):
+        self._settings_section_template.real_name['objectName'] = f'{index}-{section_name}MenuItem'
         self._settings_section_template.click()
 
     @allure.step('Open messaging settings')
     def open_messaging_settings(self) -> 'MessagingSettingsView':
-        self._open_settings(3)
+        self._open_settings(3, 'App')
```
Comment (StateOf-Vlado) [inline gui/screens/settings.py:39]: Can we have the situation with the same indexes but different sections? For now, I see that all indexes unique.

Comment (Valentina1133) [reply gui/screens/settings.py:39]: Yes, we do. There are not only different indexes in real names for settings items, but also names for items, like AppMenuItem, MainMenuitem and several more

File: gui/screens/settings.py [hunk 2]
```diff
@@ -24,27 +28,37 @@ class LeftPanel(QObject):
 
     def __init__(self):
         super().__init__('mainWindow_LeftTabView')
-        self._settings_section_template = QObject('scrollView_AppMenuItem_StatusNavigationListItem')
+        self._settings_section_template = QObject('scrollView_MenuItem_StatusNavigationListItem')
 
-    def _open_settings(self, index: int):
-        self._settings_section_template.real_name['objectName'] = f'{index}-AppMenuItem'
+    def _open_settings(self, index: int, section_name: str):
+        self._settings_section_template.real_name['objectName'] = f'{index}-{section_name}MenuItem'
```
[the remark this answers is not in the document]
Comment (Valentina1133) [reply gui/screens/settings.py:34]: Indexes are unique, but menu items names inside of objectName are different, maybe I don't understand what you are saying

[the remark this answers is not in the document]
Comment (Valentina1133) [reply gui/screens/settings.py]: Discussed on a call, I finally understood how it should work.

Fixed @StateOf-Vlado

**Review by StateOf-Vlado — APPROVED** [review]

**Pull request merged**